charts/karmada/templates/post-install-job.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if semverCompare "<1.23.0-0" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
any particular reason why this was added here?

Reply:
We don't need this Job for K8s version higher than 1.23, the Job will be cleaned automatically by `ttlSecondsAfterFinished`. Please read the description of this PR.

Reply:
> This Job is no longer rendering for me

Do you mean it can't be installed on your side?

Reply:
It is no longer rendered.

The explanation makes sense. I didn't realize post-install was split into static-resources and post-install, essentially making the latter useless if using ttlSecondsAfterFinished.

Reply:
Is there anything we need to do regarding this?

By the way, I pined you on the slack @a7i about the chart ownership thing :)

Reply:
Amazing ❤️